src/components/ProductOrder/ProductOrder.js

```diff
@@ -31,11 +32,11 @@ const ProductOrder = (props) => {
       </div>
       <div className={`${CN}__buttons-wrapper`}>
         <Link to="/not_found">
-          <Button customClass={`${CN}__cart-btn`}>add to cart</Button>
+          <Button onClick={handleClick()} className={`${CN}__cart-btn`}>add to cart</Button>
```

Comment:
`onClick={handleClick()}` - this is a bug. handleClick should be without `()` otherwise you are calling it 2 times